docs/monitoring/03-02-istio.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Istio Monitoring
+type: Details
+---
+
+The monitoring chart is pre-configured to collect all metrics relevant for observing the in-cluster [Istio](https://istio.io/latest/docs/concepts/observability/) service mesh, including the proxy-level, service-level and control-plane metrics.
+
+The concept for collecting the [service-level](https://istio.io/latest/docs/concepts/observability/#service-level-metrics) metrics is build around the Istio Proxy (implemented by Envoy), which is counting decentralized all communication inside the service mesh. After scraping these high cardinality metrics from the envoys, still an aggregation on service level is required to get the final service related metrics.
```

Comment:
```suggestion
The concept of collecting the [service-level](https://istio.io/latest/docs/concepts/observability/#service-level-metrics) metrics is built around the Istio Proxy implemented by Envoy. Istio Proxy collects all decentralized communication details inside the service mesh. After scraping these high cardinality metrics from the envoys, the metrics need to be additionally aggregated on a service level to get the final service-related details.
```

Reply:
Istio Proxy collects decentralized all communication details

Reply:
I adjusted your comment to easily merge it, please review